### PR TITLE
Feature/remove fastrtpsgen [6480]

### PR DESCRIPTION
**Pull request by LuisGP**
Backport of #645 + #677 
* Remove FastRTPSGen subproject.
* Adds `fastrpts.repos` file.
* Adapted to new installer script.

### CODE REVIEW

File: fastrtps.repos [hunk 1]
```diff
@@ -0,0 +1,18 @@
+repositories:
+    fastcdr:
+        type: git
+        url: https://github.com/eProsima/Fast-CDR.git
+        version: master
+    fastrtps:
+        type: git
+        url: https://github.com/eProsima/Fast-RTPS.git
+        version: 1.8.2
```
Comment (raquelalvarezbanos) [inline fastrtps.repos]: We should add version numbers to the other repositories too.

Comment (LuisGP) [reply fastrtps.repos]: I guess the other repositories should be at their current latest, right?

Comment (raquelalvarezbanos) [reply fastrtps.repos]: Release v1.9.1 had then with the explicit versions, check here: https://github.com/eProsima/Fast-RTPS/blob/1.9.x/fastrtps.repos

**Comment by raquelalvarezbanos** [comment]
Windows and Linux passed (see http://ci.intranet.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/529/ and http://ci.intranet.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/533/ respectively). As this PR removes `fastrtpsgen` from the repo and makes some changes to our `cmake`, I am not going to wait for Mac.

**Pull request merged**